apps/condo/migrations/20231124091332-0343_remove_b2capp_shortdescription_and_more.js

```diff
@@ -0,0 +1,34 @@
+// auto generated by kmigrator
+// KMIGRATOR:0343_remove_b2capp_shortdescription_and_more:IyBHZW5lcmF0ZWQgYnkgRGphbmdvIDQuMS41IG9uIDIwMjMtMTEtMjQgMDQ6MTMKCmZyb20gZGphbmdvLmRiIGltcG9ydCBtaWdyYXRpb25zCgoKY2xhc3MgTWlncmF0aW9uKG1pZ3JhdGlvbnMuTWlncmF0aW9uKToKCiAgICBkZXBlbmRlbmNpZXMgPSBbCiAgICAgICAgKCdfZGphbmdvX3NjaGVtYScsICcwMzQyX2F1dG9fMjAyMzExMTRfMDcyMCcpLAogICAgXQoKICAgIG9wZXJhdGlvbnMgPSBbCiAgICAgICAgbWlncmF0aW9ucy5SZW1vdmVGaWVsZCgKICAgICAgICAgICAgbW9kZWxfbmFtZT0nYjJjYXBwJywKICAgICAgICAgICAgbmFtZT0nc2hvcnREZXNjcmlwdGlvbicsCiAgICAgICAgKSwKICAgICAgICBtaWdyYXRpb25zLlJlbW92ZUZpZWxkKAogICAgICAgICAgICBtb2RlbF9uYW1lPSdiMmNhcHBoaXN0b3J5cmVjb3JkJywKICAgICAgICAgICAgbmFtZT0nc2hvcnREZXNjcmlwdGlvbicsCiAgICAgICAgKSwKICAgIF0K
+
+exports.up = async (knex) => {
+    await knex.raw(`
+    BEGIN;
+--
+-- Remove field shortDescription from b2capp
+--
+ALTER TABLE "B2CApp" DROP COLUMN "shortDescription" CASCADE;
+--
+-- Remove field shortDescription from b2capphistoryrecord
+--
+ALTER TABLE "B2CAppHistoryRecord" DROP COLUMN "shortDescription" CASCADE;
+COMMIT;
+
+    `)
+}
+
+exports.down = async (knex) => {
+    await knex.raw(`
+    BEGIN;
+--
+-- Remove field shortDescription from b2capphistoryrecord
+--
+ALTER TABLE "B2CAppHistoryRecord" ADD COLUMN "shortDescription" text NULL;
+--
+-- Remove field shortDescription from b2capp
+--
+ALTER TABLE "B2CApp" ADD COLUMN "shortDescription" text NOT NULL;
```

Comment:
It looks like this down-migration will not work in the case of `B2CApp` table having some records, because of "NOT NULL". Is it ok?